Homepage.py

```diff
@@ -61,22 +66,32 @@ def run_query(query):
 
 def createSentence(lista: list):
     sentence = f"('{lista[0]}'"
-    print(f'LEN:{len(lista)}')
     if len(lista) > 1:
         for c in range(1, len(lista)):
             sentence += ",'" + lista[c] + "'"
     sentence += ")"
     return sentence
 
-if buttonSearch:
-    #Funcion que retorna los paises seleccionados en forma de query
-    sentence = createSentence(country)
-    
-    rows = run_query(f"select * from DatosCovid as dc where dc.Date >= '{start_date}' and dc.Date <= '{end_date}' and dc.Location IN {sentence};")
-    # Print results.
-    for row in rows:
-        st.write(f"{row[0]} - {row[1]} - {row[2]} - {row[3]} - {row[4]}")
+def isAllCorrect() -> bool:
+    return len(country) > 0

```

Comment:
is this the only validation the app need?

Reply:
Yes, because the other fields have an initial value so if the user doesn't change any of them it won't get a error